pdcupdater/handlers/modules.py

```diff
@@ -75,6 +75,12 @@ def handle(self, pdc, msg):
 
         unreleased_variant = self.get_or_create_unreleased_variant(pdc, body)
 
+        if body['state'] == 5:
+            uid = unreleased_variant['variant_uid']
+            # This submits an HTTP PATCH.
+            # The '/' is necessary to avoid losing the body in a 301.
+            pdc['unreleasedvariants'][uid + '/'] += {'variant_uid': uid, 'active': True}
```

Comment:
Yeah, I thought that too... but in practice PDC expects the patch data to also contain the lookup_field otherwise it considers it an incomplete partial patch (or something like that...).